pkg/actions/kv_migrate_test.go

```diff
@@ -47,9 +47,14 @@ func benchmarkMigrate(runCount int, b *testing.B) {
 
 	createMigrateTestData(b, ctx, database, migrateBenchRepo, runCount)
 	kvStore := kvtest.MakeStoreByName(postgres.DriverName, kvparams.KV{Postgres: &kvparams.Postgres{ConnectionString: databaseURI}})(b, ctx)
-	buf, _ := os.CreateTemp("", "migrate")
-	defer os.Remove(buf.Name())
-	defer buf.Close()
+	buf, err := os.CreateTemp("", "migrate")
+	if err != nil {
+		b.Fatal("Create temp file for migrate", err)
+	}
+	defer func() {
+		_ = os.Remove(buf.Name())
```

Comment:
Could immediately delete. That's better if the program crashes.

Reply:
True - I follow the docs, saying delete after use. Updated the code not to use files.